query-service-impl/src/test/java/org/hypertrace/core/query/service/postgres/QueryRequestToPostgresSQLConverterTest.java

```diff
@@ -1079,7 +1079,8 @@ void testQueryWithPercentileAggregation() {
 
     assertSQLQuery(
         queryRequest,
-        "select PERCENTILETDIGEST99(duration_millis) FROM public.\"span-event-view\""
+        "select percentile_cont(0.990000) within group (order by (duration_millis) asc)"
```

Comment:
Is the name correct? 
`percentile_cont` -> `percentile_count` ?

Reply:
It is correct